data/reusables/actions/github_token-input-example.md

```diff
@@ -4,13 +4,14 @@ This example workflow uses the [labeler action](https://github.com/actions/label
 name: Pull request labeler
 on: [ pull_request_target ]
 
-permissions:
-  contents: read
-  pull-requests: write
-
 jobs:
   triage:
     runs-on: ubuntu-latest
+
```

Comment:
I wasn't sure whether to include whitespace here...

1. I find the whitespace makes the example more readable, especially since it's trying to highlight the `permissions` stanza.
2. But it's inconsistent with the other example on one of the pages where this is used: https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Personally, I'd resolve this by adding whitespace to 2️⃣ but I'm fine either way, I defer to the expertise of the docs team, so do whatever is best and merge down.